<ui version="4.0" >
 <class>QDialogCDProperties</class>
 <widget class="QDialog" name="QDialogCDProperties" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Label properties</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>3</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <layout class="QFormLayout" name="formLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Label mode:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QFrame" name="frame_2" >
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3" >
               <item>
                <widget class="QRadioButton" name="radioModeFull" >
                 <property name="text" >
                  <string>Full</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioModeTitle" >
                 <property name="text" >
                  <string>Title</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioModeContent" >
                 <property name="text" >
                  <string>Content</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnColor" >
            <property name="text" >
             <string>Change color...</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCDView" name="cdView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="interactive" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCDView</class>
   <extends>QGraphicsView</extends>
   <header>qcdview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogCDProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogCDProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
